### ISSUE TITLE
[1.16.5] Steel balancing suggestion

### ISSUE THREAD

**Issue by tanyan-tanyan** [MapperTV/mapperbase#8]
Hi! I saw the discussion about mapperbase steel in the curseforge comments section and having also experienced a similar problem with how easy the steel is to make, I want to suggest an alternative to the current recipe, as I saw you were looking to change it.

The only other steel providing mods I know of are Mekanism and Immersive Engineering, so I will just use them for contrast.
____________________________
### **How steel is made in Immersive Engineering**

**TLDR:** To make steel, you need two multiblock furnaces - one to get specialized fuel for the second multiblock furnace, from where you put iron to turn it into steel. Each multiblock furnace requires a minute to complete an operation.

___

Immersive Engineering has its own multiblock blast furnace which takes about 1 minute for the iron ingot to turn into a steel ingot, however the fuel it needs to do this is not coal, it's coal coke or charcoal. The coal coke can be made by putting coal in a coke oven, another IE furnace. The coke oven will then take about a minute to make coal coke and will create creosote oil on the side.

![javaw_UKWPpfHhQn](https://user-images.githubusercontent.com/68793071/121550852-0573f780-ca07-11eb-8547-8b4de4e315e8.png)

![javaw_DcucNxTgNE](https://user-images.githubusercontent.com/68793071/121550926-19b7f480-ca07-11eb-8639-b2de937fbe18.png)

![javaw_hIlYxAlgIz](https://user-images.githubusercontent.com/68793071/121560514-7f0fe380-ca0f-11eb-9fee-d86edd9f2ee0.png)

____________________________
### **How steel is made in Mekanism**

**TLDR:** To make steel, you need RF/FE (power) and 1 machine (2 machines to make the process more efficient and take up less fuel), and coal or other carbon fuel. Coal gets turned into liquid state for the use of one of the machines.

___

Mekanism uses a metallurgic infuser machine to make steel. The way it works, you insert a specific type of infusion (such as redstone, coal, etc) and it fills up the tank of the machine. For efficiency purposes, you would put 1 coal into a machine called enrichment chamber to get 1 enriched carbon. That way, when you feed the enriched carbon to the infuser, it fills up more of the tank than it would with raw coal. This means that 1 raw coal will be enough for 1 infusion, whereas 1 enriched carbon (1 processed coal) will be enough for 8 infusions, etc. The mekanism steel recipe requires you to turn iron into enriched iron, and then enriched iron into steel dust using carbon infusion.
Additionally, you need to power all of the mekanism machines to produce steel which is an extra requirement.

![javaw_SOOowUWyRc](https://user-images.githubusercontent.com/68793071/121565145-1b3be980-ca14-11eb-90cd-82c0c94bf478.png)

![javaw_I1w5OYbPsc](https://user-images.githubusercontent.com/68793071/121565386-4fafa580-ca14-11eb-909f-02176d065793.png)

![javaw_HjCbcV5mPi](https://user-images.githubusercontent.com/68793071/121565540-6e15a100-ca14-11eb-89b3-2639a6a165c6.png)

![javaw_gqYz1JeAcv](https://user-images.githubusercontent.com/68793071/121565576-7968cc80-ca14-11eb-838a-5489968c7769.png)

_____
### **Suggestion for MapperBase**

1. I'm not sure if you can explicitly require the steel recipe to require bitumen instead of coal in the blast furnace, but if it is possible, why not use bitumen to make steel? Should it end up being the case and you want to use it for the new steel recipe, you could also kill 2 birds with 1 stone by significantly reducing the default bitumen cluster size. I know config is a thing, but I'll be honest - current bitumen size is too overwhelming and we shouldn't have to change the config every time we add MapperBase to the mod list.

2. This is an uneducated theory - I believe it is possible to increase the time it takes for the blast furnace to complete an operation as a friend of mine made a blast furnace CraftTweaker script for another item, and there was a cooking time configuration. This could greatly nerf your mod's steel production, but it might still not be quite enough.

3. Turn 1 iron into 1 steel ingot - the simplest approach in my opinion. I doubt option 1 is possible and I'm not sure if option 2 is going to be enough, but option 3 might be just enough to balance it with other steel production methods. It might seem like a pretty harsh deal, but steel is supposed to be valuable and a bit harder to make.

What do you think?

**Comment by MapperTV** [comment]
I already increased the duration in the past which increase coal consumption but people still though it was too easy. So I switched the recipe to 1 iron ingot -> 1 steel nugget (same for iron block to steel ingot), it will be applied in the next update which is technically already done, just not published yet as I upload all my mods updates at once.

The other change is reducing the steel gear efficiency and durability so it is closer to iron than diamond (basically similar to iron but with a better durability).

As for bitumen, next update will have slightly bigger pockets, but less common. The height is also reduced to around water level. And biomes that are mountainous won't have bitumen anymore. Still not perfect but should improve the worldgen a bit. In future version I want to attempt a new biome where bitumen is abundant and other biomes will have way less of it.

**Comment by tanyan-tanyan** [comment]
Ohhh. That's really nice to hear, thank you for such a quick response!

**Comment by MapperTV** [comment]
New version is now released with a completely new way to craft steel.

Steps are:

1. blast iron ingot into ferrite (consumes one coal)
2. craft raw steel with 3 ferrite
3. blast raw dust into steel ingot (consumes ½ coal)

This technically only requires one vanilla blast furnace however it is not that efficient in terms of resources as one steel ingot requires 3 iron ingots and almost 4 coal. Players can decide to keep this simple setup and use more resources or switch to Mekanism / IE setups which are more complex but way more efficient.

_All of this is subject to changes depending of received feedback._
